environments/staging/inventory.ini

```diff
@@ -1,53 +1,90 @@
-[proxy]
+[proxy0-staging]
```

Comment:
why the `-staging` suffix? We don't use that anywhere else.

Reply:
I just left it to match the host name in `proxy0-staging.internal-va.commcarehq.org`. Happy to get rid of it, since I don't think there's any practical implications and it's maybe cleaner.